packages/editor/src/components/editors/InscriptionEditor.tsx

```diff
@@ -42,6 +43,16 @@ const Header = (props: EditorProps) => {
             {data.name.length > 0 && ` - ${data.name}`}
           </div>
         </div>
+
+        {editorContext.type.helpUrl !== undefined && (
```

Comment:
what is, when the helpUrl is empty ("")?
I think I would not show a help button if it is like this...